python/modem-firmware-1.3+/nrf_cloud_device_mgmt.py

```diff
@@ -682,7 +768,7 @@ def main():
         update_by = updateBy.DEV_ID
     else:
         print('Getting all devices...')
-        if args.tag_list or args.tag:
+        if args.tag_list or args.tag or fota_type != updateBundle.fotaType.MODEM:
```

Comment:
Is the lack of a check for `MDM_FULL` intentional, here? And, actually, what is the purpose of this check?

Reply:
not intentional.
`"Display all tags (device groups) and prompt to select tag to use. Enabled for non-MODEM updates.",`